### ISSUE TITLE
Animeflix

### ISSUE THREAD

**Issue by joessef45** [aniyomiorg/aniyomi-extensions#1954]
### Source name

AnimeFlix

### Source link

http://animeflix.live

### Source language

en

### Other details

I wish that you could add this site to the app, It has **no** hentai.
 it has many servers some of them have multi sub and also some of them uses crunchy an the other domain is animeflix.icu
 Thank you!

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://aniyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/aniyomiorg/aniyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

**Comment by Fidelo8** [comment]
Would be good, current extension isn't even working...

**Comment by Secozzi** [comment]
> Would be good, current extension isn't even working...

What do mean by "current extension" since there isn't any extension for animeflix.live?

**Comment by Fidelo8** [comment]
Oh it's not the same site, but same name...

**Comment by Secozzi** [comment]
Yes they're two unrelated sites that just so happen to share the same name. If you have any problems with the existing animeflix, then create a new issue about it.
